.env.sample

```diff
@@ -50,7 +50,9 @@ UDW_PORT=
 UDW_ID_PREFIX=
 
 # Canvas Configuration
+# put course canvas ids here, separated by ","
 CANVAS_COURSE_IDS=
+CANVAS_COURSE_IDS=COURSE_ID_1,COURSE_ID_2
```

Comment:
I have a WIP PR to store these in a database table, this is fine for now though. I'll change it next week.